recipe/meta.yaml

```diff
@@ -10,8 +10,9 @@ source:
     md5: 741907c4e36923601827ad06d9ba7bf1
 
 build:
-    number: 0
-    script: python setup.py install --single-version-externally-managed --record=record.txt
+  noarch: python
+    number: 1
+    script: python -m pip install --no-deps --ignore-installed .
```

Comment:
It did not add `pip` :-/

Maybe we should just trigger the `add noarch` webservice and add `pip` with another commit from the bot to separate possible the issues.